src/setuptools_scm/integration.py

```diff
@@ -9,6 +9,31 @@
 from .utils import trace
 
 
+def _break_on_old_setuptools(_version=setuptools.__version__):
+    if int(_version.split(".")[0]) < 45:
+        raise SystemExit(
+            f"""
+ERROR: setuptools=={_version} is used in combination with setuptools_scm>=6.x
+
+Your build configuration is incomplete and worked by accident!
+Please ensure setuptools>=45 and setuptools_scm>=6.2 are installed with current tools.
```

Comment:
This sentence is confusing to me. I think you mean to say that we should use PEP 517 tooling, which provisions the environment by default, right? If so,

```suggestion
Please ensure setuptools>=45 and setuptools_scm>=6.2 are built with recent tooling.
```

Even then, I am no so sure this is sentence is clear. Perhaps it would be best to just remove it in favor of the last point of the list below?

Reply:
```suggestion
```